src/main/java/com/gmail/yuramitryahin/modal/Driver.java

```diff
@@ -40,6 +50,22 @@ public void setLicenseNumber(String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
```

Comment:
if we'll get to compare two Driver objects, what we would get?

Reply:
Why do we need to compare two drivers?

Reply:
we can compare by id if we need